assets/js/highscores.js

```diff
@@ -0,0 +1,32 @@
+const clearScoresBtn = document.querySelector("#clear-scores");
+const highScoresList = document.querySelector("#high-scores-list");
+
+// get scores/initials from LS (array of objects)
+const getScoreData = function () {
+  const highScoresFromLS = JSON.parse(localStorage.getItem("highScores"));
```

Comment:
As mentioned on the feedback, there are a lot of function declarations, which is fine and they do work, consider using ES6/Arrow functions next time you do this as these are more commonly used. 

`const highScoresFromLS = () => { CODE HERE }`